.circleci/config.yml

```diff
@@ -450,10 +473,34 @@ jobs:
           command: |
             cd server
             yarn test --coverage
-            cd ..
       - store_artifacts:
           path: server/coverage
 
+# Publish the android app
+  publish-android-app-appcenter:
+    executor:
+      <<: *android-build
+    parameters:
+      distribution-audience:
+        # Use this flag to control the app distrubution audience
+        type: string
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: "Publishing Android app to AppCenter"
+          command: |
+            npm install webpack
+            echo no | npm install -g appcenter-cli
```

Comment:
Why do we need to perform these steps? `webpack` is not installed globally here and pretty much won't do anything, unless I'm missing something...

Reply:
I guess we don't need it, I had it because in other build scripts we have, we also installed it prior to installing app-center-cli

Reply:
I took a look at the appcenter CLI docs and it doesn't seem to need webpack.